pkg/kb-summarizer/summarizer/summarizer.go

```diff
@@ -638,6 +641,19 @@ func (s *Summarizer) runFinalPassOnCheckWrapper(cw *CheckWrapper) {
 	}
 }
 
+func (s *Summarizer) copyDataFromResults(cw *CheckWrapper) {
+	checkFromResults := s.checkWrappersMaps[cw.ID]
+	if checkFromResults == nil {
+		return
+	}
+	cw.Audit = checkFromResults.Audit
+	cw.AuditConfig = checkFromResults.AuditConfig
+	cw.ActualValue = checkFromResults.ActualValue
```

Comment:
If the same check runs on two nodes with a pass on one and a fail on the other one, actualValue would be overwritten, right?

Reply:
oh, so then should be a map <nodeIP>:<value>?

Reply:
For interpolation of the variables, it's ok to copy over from the result. But the "actual value" is per node, so we might have to store using a map.

Reply:
Added a map